src/bioregistry/data/bioregistry.json

```diff
@@ -42077,6 +42077,63 @@
     },
     "uri_format": "https://www.ncbi.nlm.nih.gov/nucleotide/$1"
   },
+  "genbase": {
+    "contact": {
+      "email": "genbase@big.ac.cn",
+      "name": "GenBase"
```

Comment:
We should use the person listed under fairsharing as the contact since the convention in Bioregistry is to use a specific person rather than a generic contact address.